Tests/README.md

```diff
@@ -0,0 +1,33 @@
+# Testsuite for SCS standards
+
+The tool `scs-compliance-check.py` parses a 
+[compliance definition file](https://github.com/SovereignCloudStack/standards/pull/214#issuecomment-1429883879)
+and executes the test executables referenced in there for
+the specified layer (`iaas` or `kaas`).
+
+## Local execution (Linux, BSD, ...)
+
+On your Linux machine, ensure you have python3-openstacksdk and
+python3-PyYAML installed, a cloud environment configured in your
+`~/.config/openstack/clouds.yaml` and `secure.yaml` and then run
```

Comment:
The requirements can be installed by invoking
```
pip install -r requirements.txt
```
Should we highlight this here?

Reply:
Well, yes, if people want to go the pip route.
I personally use distributions that have working distro packages.
So, I would suggest that we mention both possibilities.

Reply:
Unrelated: you don't need to have a `secure.yaml` file. That being said, I guess you cannot describe every method (installations or config files). I would suggest picking one way and improve over time if required by consumers.

Reply:
@tibeer: It is best practice to split out secrets into a well-protected `secure.yaml`, that's why I mention it.
If users know that they can actually consolidate everything in `clouds.yaml` and want to do that, so be it.
It's not what I consider a good default expectation, thus referencing `secure.yaml`.